### PR TITLE
Updating notebooks and small issues. 

### PR DESCRIPTION
**Pull request by czewski**
This PR introduces some updates related to the issue #17, including updates to all notebooks, and some small fixes that will be specified below. 

1. Fixed images in the README.md
2. I have not found the duplicates of .gitignore (maybe we already merged it) 
3. Package installation had error while importing relative README.md
4. Notebook updates: 
    - notebooks\assessment_examples.ipynb. - ok
    - notebooks\dataset_examples.ipynb. - works fine for us, but we made a update to point to the Dataset inside the repo, so the function does not try to download the dataset again. 
    - notebooks\feature_extraction_examples.ipynb --> fixed
    - notebooks\load_model_examples.ipynb. --> The errors in this notebook are under the section "Error handling patterns", they are expected.  
    - notebooks\mlp_timeseries_example.ipynb. --> Update the reference to the Dataset directory. 
    - notebooks\plot_series_examples.ipynb. --> Fixed. 
    - notebooks\sklearn_models_examples.ipynb. --> Fixed
    - notebooks base.ipynb --> Removed as it was empty. 
   
 5. Set all _dataset_path_ to "_../../dataset_"

 **Updates still needed**: 

1. We should not need "_sys.path.append(os.path.abspath(os.path.join(os.getcwd(), "../../")))_", as we are setting a requirement for the user to install the toolkit properly. 

2. Notebooks to update: (Use 3W dataset, Add more comments)- load_model_examples.ipynb --> Update documentation
- **report_generation_examples**.ipynb. --> It has a problem related to not founding the correct template for the HTML report. Should be updated. 
- **train_test_split_examples**.ipynb --> Suggestion: Remove notebook, as the Toolkit does not have this function anymore (just a method inside the trainer). 
	- BaseTimeSeriesHoldout has nothing. 
- **feature_extraction_examples**.ipynb --> Exponentially Weighted Statistical Feature Extraction takes more than 40min to run, something may be wrong. 
- **common_preprocessing**.ipynb - I made half of this notebook work, but somebody have to update it, adding relevant comments. 
	- The ParquetDataset class does not have the transform method anymore, is there a way to still use default_data_normalization, default_data_processing?
	- I saw that we have Normalize() method, maybe we should remove the methods mentioned above?

### CODE REVIEW

**Review by rafaelpadilla — APPROVED** [review]
Thank you!

I just added a coupe commits to solve some merging conflicts!
